app/src/main/java/com/github/sdpteam15/polyevents/view/activity/activityprovider/MyItemRequestsActivity.kt

```diff
@@ -127,7 +131,10 @@ class MyItemRequestsActivity : AppCompatActivity(), AdapterView.OnItemSelectedLi
         }
 
         requests.group(this, materialRequest) {
-            it.status
+            when (it.status) {
+                DELIVERING, DELIVERED, RETURN_REQUESTED, RETURNING, RETURNED -> ACCEPTED
+                else -> it.status
+            }
```

Comment:
Nice, this way we don't have 10 millions different pages for item requests

Reply:
yes, I found I think it is better for the admins who see all the requests from everybody